backend/Makefile

```
@@ -1,5 +1,8 @@
 .PHONY: build test test-integration lint clean swag seed
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)